internal/jasmine_node_test.bzl

```diff
@@ -9,7 +9,8 @@ def jasmine_node_test(name, srcs, data = [], args = [], deps = [], **kwargs):
   )
 
   args = ["/".join([p
-      for p in [".", PACKAGE_NAME, "%s_devmode_srcs.MF" % name]
+      for p in [".", "external" if REPOSITORY_NAME else "", REPOSITORY_NAME[1:],
```

Comment:
better to show what character is being stripped, either add a comment, or maybe
`REPOSITORY_NAME[len("."):]` to show which prefix we expect to be removed
or both
(isn't this a bug though, don't you want to split on `/` and take the second part?)

Reply:
Will change to use `len("@")`.
`REPOSITORY_NAME` is something like `@angular`.